ui/src/dashboards/components/DashboardPage.tsx

```diff
@@ -156,20 +111,21 @@ class DashboardPage extends Component<Props> {
       children,
     } = this.props
 
+    const {dashboardID} = params
+
     return (
       <Page titleTag={this.pageTitle}>
         <LimitChecker>
           <HoverTimeProvider>
             <DashboardHeader
-              org={org}
-              dashboard={dashboard}
+              orgName={orgName}
```

Comment:
not a big dealm, but shouldn't this component grab its own data?

Reply:
it can.  and i think it just might!